tg/Favorites/View/FavoritesViewController.swift

```diff
@@ -52,8 +55,8 @@ extension FavoritesViewController: TableListFavoritesOfGameCellDelegate {
     func alertDelete(_ id: Int) {
         let alert = UIAlertController(title: "Delete this game?", message: nil, preferredStyle: .alert)
         let actionOkey = UIAlertAction(title: "Okey", style: .default) { ACTION in
-            CoreDataManager.dataManager.deleteItem(id: id)
-            self.arrayOfAddedGames = CoreDataManager.dataManager.receiveData()
+            self.presenter.tableDeleteCell(id: id)
+            self.presenter.reloadListOfGames()
             self.tableFavorites.reloadData()
```

Comment:
It isn't a good idea to reload the table and data after each deletion. You can manually delete elements from the array and tableView